alibi_detect/saving/tests/test_saving.py

```diff
@@ -593,7 +921,7 @@ def test_load_absolute(data, tmp_path):
     # Write a new cfg file elsewhere, with x_ref reference inside it an absolute path to original x_ref location
     cfg = read_config(tmp_path.joinpath('config.toml'))
     x_ref_path = tmp_path.joinpath(Path(cfg['x_ref'])).resolve()  # Absolute path for x_ref
-    cfg['x_ref'] = x_ref_path
+    cfg['x_ref'] = x_ref_path.as_posix()  # we always write paths to config.toml as Posix not Windows paths
```

Comment:
Seems like this should be documented in the main saving implementation (also refer to the comment above about the one use of `as_posix` which I found - should there be more?).

Reply:
Have added a note to the docs:
![image](https://user-images.githubusercontent.com/32061685/169844749-720f0791-d322-4840-a71d-893e3d0c2d3e.png)

See https://github.com/SeldonIO/alibi-detect/pull/497#discussion_r879426796 regarding `as_posix`.